### PR TITLE
Desktop: Update external link whitelist

### PR DESCRIPTION
**Pull request by rihardsgravis**
# Description

Update external link whitelist to match all *Terms & Conditions* and *Privacy policy* included links

## Type of change

- Enhancement (a non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines for this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: src/desktop/main.js [hunk 1]
```diff
@@ -216,7 +216,16 @@ function createWindow() {
...
             e.preventDefault();
 
-            const externalWhitelist = ['iota.org', 'docs.iota.works', 'trinity.iota.org', 'docs.bugsnag.com'];
+            const termsAndConditionsLinks = ['iota.org', 'trinity.iota.org', 'mailto:contact@iota.org'];
+            const privacyPolicyLinks = [
+                'corp.sogou.com',
+                'cryptocompare.com',
+                'docs.bugsnag.com',
+                'help.github.com',
+                'protect-eu.mimecast.com',
```
Comment (rajivshah3) [inline src/desktop/main.js:226]: Does `policies.google.com` need to be added as well?

File: src/desktop/main.js [hunk 2]
```diff
@@ -216,7 +216,16 @@ function createWindow() {
         if (url.indexOf(targetURL) !== 0) {
             e.preventDefault();
 
-            const externalWhitelist = ['iota.org', 'docs.iota.works', 'trinity.iota.org', 'docs.bugsnag.com'];
+            const termsAndConditionsLinks = ['iota.org', 'trinity.iota.org', 'mailto:contact@iota.org'];
+            const privacyPolicyLinks = [
+                'corp.sogou.com',
+                'cryptocompare.com',
```
Comment (rajivshah3) [inline src/desktop/main.js:222]: Will this allow `www.cryptocompare.com`? That seems to be the URL in the privacy policy

Comment (rihardsgravis) [reply src/desktop/main.js:222]: Yes, `www` subdomains are handled here https://github.com/iotaledger/trinity-wallet/blob/ff6dc0f60a50f2590c6019b47991f543a4bf443d/src/desktop/main.js#L232

File: src/desktop/main.js (hunk 1 above)
Comment (rihardsgravis) [reply src/desktop/main.js:226]: Added

File: src/desktop/main.js [hunk 3]
```diff
@@ -216,7 +216,17 @@ function createWindow() {
...
+                'docs.bugsnag.com',
+                'help.github.com',
+                'policies.google.com',
+                'protect-eu.mimecast.com',
```
Comment (rajivshah3) [inline src/desktop/main.js:226]: Just noticed one more, sorry. I think `mailto:privacy@iota.org` also needs to be whitelisted

**Review by rajivshah3 — APPROVED** [review]

**Pull request merged**